packages/sync-repo-settings/package.json

```diff
@@ -18,7 +18,7 @@
     "probot-app"
   ],
   "scripts": {
-    "compile": "tsc -p . && cp ./src/required-checks.json ./build/src/required-checks.json.json",
+    "compile": "tsc -p . && cp ./src/required-checks.json ./build/src/required-checks.json.json && cp ./src/teams.json ./build/src/teams.json.json",
```

Comment:
Why do we need to copy these json files at all into the build?  We could include them in the module via the `files` field in `package.json`, and rewrite the paths so we don't need these gymnastics.

Reply:
I'm not sure why this is here, but I followed the pattern.

Reply:
Filed https://github.com/googleapis/repo-automation-bots/issues/603 to track. I'm not sure how to fix this.